eng/generate-compiler-code.ps1

```diff
@@ -84,53 +84,32 @@ function Run-GetText() {
     }
 }
 
-# Build all of the tools that we need to generate the syntax trees and ensure
-# they are in a published / runnable state.
-function Build-Tools() {
-    $list = @(
-        'boundTreeGenerator;BoundTreeGenerator;BoundTreeGenerator\CompilersBoundTreeGenerator.csproj',
-        'csharpErrorFactsGenerator;CSharpErrorFactsGenerator;CSharpErrorFactsGenerator\CSharpErrorFactsGenerator.csproj',
-        'csharpSyntaxGenerator;CSharpSyntaxGenerator;CSharpSyntaxGenerator\CSharpSyntaxGenerator.csproj',
-        'basicErrorFactsGenerator;VBErrorFactsGenerator;VisualBasicErrorFactsGenerator\VisualBasicErrorFactsGenerator.vbproj',
-        'basicSyntaxGenerator;VBSyntaxGenerator;VisualBasicSyntaxGenerator\VisualBasicSyntaxGenerator.vbproj')
-
-    $dotnet = Ensure-DotnetSdk
-    $tfm = "netcoreapp2.1"
-    $rid = "win-x64"
-
-    $sourceDir = Join-Path $RepoRoot 'src\Tools\Source\CompilerGeneratorTools\Source'
-    foreach ($item in $list) { 
-        $all = $item.Split(';')
-        $varName = $all[0]
-        $exeName = $all[1]
-        $proj = Join-Path $sourceDir $all[2]
-        $fileName = [IO.Path]::GetFileNameWithoutExtension($proj)
-        Write-Host "Building $fileName"
-
-        Exec-Command $dotnet "publish /p:Configuration=$configuration /p:RuntimeIdentifier=$rid /v:m `"$proj`"" | Out-Null
```

Comment:
I honestly can't remember exactly why I did `publish` in the first place. I imagine it was because I was trying to isolate out the phases here: build and execute. This code comes from the early days of CLI where we were still figuring it all out. Now though we can use `dontnet run` and just let its thing.